### PR TITLE
Completed task/236-colors-refactoring

### PR DESCRIPTION
**Pull request by TomasPetrik**
Show darkmode switch in settings for test build only =>
"--dart-define=IS_PROD=false" will appear switch.

### CODE REVIEW

File: lib/view/screens/asset_receive_screen.dart [hunk 1]
```diff
@@ -46,7 +48,7 @@ class AssetReceiveScreen extends StatelessWidget with WalletUtilsMixin {
         final assetDetail = Assets.getAssetDetail(assetWallet.assetId);
         final networkDetail = Assets.getNetworkDetail(networkWallet.networkId);
 
-        return Scaffold(
+        return SioScaffold(
```
[the remark this answers is not in the document]
Comment (TomasPetrik) [reply lib/view/screens/asset_receive_screen.dart:51]: Fixed and re-designed.

File: lib/main.dart [hunk 2]
```diff
@@ -84,11 +85,13 @@ class _SimplioAppState extends State<SimplioApp> {
         )..add(GotLastAuthenticated()),
         child: AuthGuard(
           onAuthenticated: (context, authState) {
+            isAuthentificated = true;
```
[the remark this answers is not in the document]
Comment (TomasPetrik) [reply lib/main.dart:88]: Fixed.

File: lib/view/screens/splash_screen.dart [hunk 3]
```diff
@@ -2,8 +2,9 @@ import 'package:flutter/material.dart';
 import 'package:flutter/services.dart';
 import 'package:simplio_app/data/model/account_settings.dart';
 import 'package:simplio_app/view/themes/constants.dart';
-import 'package:simplio_app/view/themes/dark_mode.dart';
-import 'package:simplio_app/view/themes/light_mode.dart';
+import 'package:simplio_app/view/themes/sio_colors.dart';
```
[the remark this answers is not in the document]
Comment (TomasPetrik) [reply lib/view/screens/splash_screen.dart:5]: This screen is gonna be redesigned anyway - by using dark mode only.

File: lib/view/screens/asset_exchange_screen.dart [hunk 4]
```diff
@@ -3,6 +3,7 @@ import 'package:flutter/material.dart';
 import 'package:flutter_bloc/flutter_bloc.dart';
 import 'package:go_router/go_router.dart';
 import 'package:simplio_app/l10n/localized_build_context_extension.dart';
+import 'package:simplio_app/logic/cubit/account_wallet/account_wallet_cubit.dart';
```
[the remark this answers is not in the document]
Comment (TomasPetrik) [reply lib/view/screens/asset_exchange_screen.dart:6]: Fixed.

File: lib/view/screens/asset_exchange_summary_screen.dart [hunk 5]
```diff
@@ -8,6 +8,7 @@ import 'package:simplio_app/view/routes/authenticated_router.dart';
 import 'package:simplio_app/view/screens/mixins/wallet_utils_mixin.dart';
 import 'package:simplio_app/view/themes/constants.dart';
 import 'package:simplio_app/view/themes/simplio_text_styles.dart';
+import 'package:simplio_app/view/themes/sio_colors.dart';
```
[the remark this answers is not in the document]
Comment (TomasPetrik) [reply lib/view/screens/asset_exchange_summary_screen.dart:11]: Fixed.

File: lib/view/themes/simplio_text_styles.dart [hunk 6]
```diff
@@ -9,23 +9,27 @@ class SioTextStyles {
   static const double _letterSpacing2 = 0.2;
   static const double _letterSpacing3 = 0.3;
 
-  static TextStyle buttonStyle = const TextStyle(
+  static TextStyle sioTextStyle = const TextStyle(
+      //color: SioColors.softBlack,
```
[the remark this answers is not in the document]
Comment (TomasPetrik) [reply lib/view/themes/simplio_text_styles.dart]: I removed this.

**Review by jvrastil — APPROVED** [review]

**Pull request merged**